File: Patterns/Pattern-20.cpp
/*--

*				*	
*				*	
*		*		*	
*	*		*	*	
*				*	

--*/
//code
#include <iostream>
using namespace std;
int main(int argc, char**argv){
    int n;
    cin >> n;
    for(int r=1;r<=n;r++){
        for(int c=1;c<=n;c++){
            if(c==1 || c==n){
                cout <<"*\t";
            }
            else if(r==c && r>(n/2)){
                cout << "*\t";
            }
            else if((r+c==n+1) && (r>=(n/2)+1)){
                cout <<"*\t";
            }
            else{
                cout <<"\t";
            }
            
        }
            
        
        cout <<"\n";
    }

    //write your code here

}

File: Patterns/java/Pattern-20.java
/*--

*				*	
*				*	
*		*		*	
*	*		*	*	
*				*	

--*/
//code
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n=scn.nextInt();
        for(int r=1;r<=n;r++){
            for(int c=1;c<=n;c++){
                if(c==1 || c==n){
                    System.out.print("*\t");
                }
                else if(r==c && r>(n/2)){
                    System.out.print("*\t");
                }
                else if((r+c==n+1) && (r>=(n/2)+1)){
                    System.out.print("*\t");
                }
                else{
                    System.out.print("\t");
                }
                
            }
            
            System.out.println();
        }
        // write ur code here

    }
}